src/commands/monitor.rs

```diff
@@ -530,26 +543,19 @@ async fn submit_and_watch_solution<T: MinerConfig + Send + Sync + 'static>(
 	listen: Listen,
 	dry_run: bool,
 	at: &Header,
+	signed_phase_length: u64,
 ) -> Result<(), Error> {
 	let tx = epm::signed_solution(RawSolution { solution, score, round })?;
 
 	// TODO: https://github.com/paritytech/polkadot-staking-miner/issues/730
 	//
-	// The extrinsic mortality length is static and doesn't know when the
-	// signed phase ends.
-	let signed_phase_len = client
```

Comment:
Just to double check: Instead of fetching the signed phase constant from the chain, we define our own signed phase in the static types.rs. And this is because the constant from the chain wasn't what we'd expect?

Reply:
from the PR description: its because the `SignedPhase` is removed from westend in preparation for the merkalized metadata.